### PR TITLE
[SYSDM] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/cpl/sysdm/lang/ro-RO.rc [hunk 1]
```diff
@@ -34,236 +34,236 @@ END
...
-    AUTORADIOBUTTON "Calculatorul &este andocat", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
-    AUTORADIOBUTTON "Calculatorul n&u este andocat", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
-    GROUPBOX "Selecție profile de dispozitive", IDC_STATIC, 10, 138, 192, 42
-    AUTOCHECKBOX "In&clude întotdeauna acest profil ca opțiune la pornirea ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
+    AUTOCHECKBOX "Acesta este un computer &portabil", IDC_HWPPROP_PORTABLE, 17, 68, 120, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "&Stare de andocare necunoscută", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Computerul este &andocat", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
+    AUTORADIOBUTTON "Computerul este &detaşat", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
+    GROUPBOX "Secţiunea profiluri hardware", IDC_STATIC, 10, 138, 192, 42
+    AUTOCHECKBOX "Întotdeauna &include acest profil ca opţiune la pornirea ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/ro-RO.rc:129]: pic please of ro-RO MS dlg for the &i accelerator.

File: dll/cpl/sysdm/lang/ro-RO.rc [hunk 2]
```diff
@@ -34,236 +34,236 @@ END
...
             UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS | WS_CHILD | WS_VISIBLE, 0, 0, 8, 13
     LTEXT "secunde", IDC_STATIC, 209, 70, 30, 8
-    LTEXT "Puteți alege editarea manuală a opțiunilor de pornire:", IDC_STATIC, 14, 89, 187, 8
-    PUSHBUTTON "&Editare…", IDC_STRRECEDIT, 188, 87, 50, 14
-    GROUPBOX "Căderea sistemului", IDC_STATIC, 7, 111, 238, 140
-    AUTOCHECKBOX "&Consemnează evenimentul în jurnalul de sistem", IDC_STRRECWRITEEVENT, 14, 124, 178, 10
-    AUTOCHECKBOX "T&ransmite alertă administrativă", IDC_STRRECSENDALERT, 14, 138, 148, 10
-    AUTOCHECKBOX "Repornește &automat sistemul", IDC_STRRECRESTART, 14, 152, 145, 10
-    GROUPBOX "Scriere informații de depanare", IDC_STATIC, 12, 167, 227, 76
+    LTEXT "Pentru a edita fişierul cu opţiuni de pornire, faceţi clic pe Editare.", IDC_STATIC, 14, 89, 187, 8
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/ro-RO.rc:222]: This line really ends with a `.` and not with a `:` ?

File: dll/cpl/sysdm/lang/ro-RO.rc [hunk 3]
```diff
@@ -34,236 +34,236 @@ END
...
-    LTEXT "Minimă permisă:", IDC_STATIC, 18, 188, 58, 9
+    PUSHBUTTON "&Stabilire", IDC_SET, 158, 154, 50, 15
+    GROUPBOX "Dimensiune totală fişier de paginare pentru toate unităţile", IDC_TOTALGROUP, 10, 177, 204, 46
+    LTEXT "Minim permis:", IDC_STATIC, 18, 188, 58, 9
     LTEXT "%s", IDC_MINIMUM, 88, 188, 100, 9
-    LTEXT "Recomandată:", IDC_STATIC, 18, 199, 52, 9
+    LTEXT "Recomandat:", IDC_STATIC, 18, 199, 52, 9
     LTEXT "%s", IDC_RECOMMENDED, 88, 199, 100, 9
-    LTEXT "Actuală:", IDC_STATIC, 18, 210, 66, 9
+    LTEXT "Alocat în prezent:", IDC_STATIC, 18, 210, 66, 9
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/ro-RO.rc:263]: Please attach a pic that this does not overflow the 66 width.

File: dll/cpl/sysdm/lang/ro-RO.rc [hunk 4]
```diff
@@ -34,236 +34,236 @@ END
...
     LTEXT "%s", IDC_DRIVE, 88, 80, 100, 9
-    LTEXT "Spațiu disponibil:", IDC_STATIC, 20, 92, 56, 9
+    LTEXT "Spaţiu disponibil:", IDC_STATIC, 20, 92, 56, 9
     LTEXT "%s", IDC_SPACEAVAIL, 88, 92, 104, 9
-    LTEXT "Di&mensiunea inițială (Mo):", -1, 32, 118, 87, 9
-    LTEXT "Dimensiunea ma&ximă (Mo):", -1, 32, 131, 87, 9
-    AUTORADIOBUTTON "Limite parti&cularizate:", IDC_CUSTOM, 20, 105, 154, 9, WS_GROUP
-    AUTORADIOBUTTON "&Limite gestionate de sistem", IDC_SYSMANSIZE, 20, 145, 188, 9
-    AUTORADIOBUTTON "Fă&ră paginare", IDC_NOPAGEFILE, 20, 158, 78,9
+    LTEXT "D&imensiune iniţială (MO):", -1, 32, 118, 87, 9
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/ro-RO.rc:250]: Ms-ro-RO-pic proving the &i accelerator please.

File: dll/cpl/sysdm/lang/ro-RO.rc [hunk 5]
```diff
@@ -34,236 +34,236 @@ END
...
     PUSHBUTTON "", IDC_HRDPROFDWN, 232, 79, 15, 14, BS_ICON | WS_DISABLED
-    PUSHBUTTON "&Proprietăți…", IDC_HRDPROFPROP, 8, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Copiere…", IDC_HRDPROFCOPY, 63, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Redenumire…", IDC_HRDPROFRENAME, 118, 106, 50, 14, WS_DISABLED
-    PUSHBUTTON "&Eliminare…", IDC_HRDPROFDEL, 173, 106, 50, 14, WS_DISABLED
-    GROUPBOX "Selecție profiluri ale dispozitivelor", IDC_STATIC, 7, 130, 240, 75
-    LTEXT "La pornirea ReactOS:", IDC_STATIC, 14, 142, 210, 8
-    AUTORADIOBUTTON "&Așteaptă alegerea explicită a unui profil", IDC_HRDPROFWAIT, 14, 158, 190, 8, WS_GROUP
-    AUTORADIOBUTTON "Alege a&utomat primul profil în lipsa unei alegeri timp de", IDC_HRDPROFSELECT, 14, 173, 190, 8
+    PUSHBUTTON "Propr&ietăţi", IDC_HRDPROFPROP, 8, 106, 50, 14, WS_DISABLED
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/ro-RO.rc:96]: Ms-ro-RO-pic proving the &i accelerator please.

**Comment by JoachimHenze** [comment]
This PR looks promising to me at first glance. A lot of stuff I do perceive as an improvement.

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 5 above)
Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/ro-RO.rc:96]: the 50 width do look like a very close fit as well. Possible that it overflows already.

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc:129]: 2024-05-05 17_56_10-Resource Hacker - sysdm cpl mui

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc:222]: Yes

2024-05-05 17_57_23-Resource Hacker - sysdm cpl mui

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 3 above)
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc:263]: 2024-05-05 17_58_12-Resource Hacker - sysdm cpl mui

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 4 above)
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc:250]: 2024-05-05 17_58_50-Resource Hacker - sysdm cpl mui

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/cpl/sysdm/lang/ro-RO.rc:96]: 2024-05-05 17_59_39-Resource Hacker - sysdm cpl mui

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 3 above)
Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/ro-RO.rc:263]: Resolved, despite the fact that this is the MS pic. But if MS doesn't overflow at 65, then we won't overflow wih 66 neither.

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 4 above)
Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/ro-RO.rc:250]: Thanks. MS uses 118 width. We do only use 87. Can you attach one more pic of the ros dlg, that proves the 87 is not causing truncation yet? Thx in advance.

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 5 above)
Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/ro-RO.rc:96]: Thanks. Resolved.

File: dll/cpl/sysdm/lang/ro-RO.rc (hunk 4 above)
Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/ro-RO.rc:250]: Resolved. Doesn't matter actually whether the 87 would overflow, because you didn't increase the strings length.

**Review by JoachimHenze — APPROVED** [review]
PR approved. Requesting review from @GeoB99 also as a native speaker.
Good job Andrei!

**Comment by AndreiMiloiu** [comment]
> PR approved. Requesting review from @GeoB99 also as a native speaker. Good job Andrei!

Thank you!

**Pull request merged**